src/js/components/dashboard/graph/BarChartXAxis.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * BarChartXAxis.jsx
+ * Created by Lizzie Salita 11/15/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import BarChartXAxisItem from './BarChartXAxisItem';
+
+const propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+    items: PropTypes.array,
+    line: PropTypes.object,
+    lineGroup: PropTypes.object,
+    labelGroup: PropTypes.object
+};
+
+export default class BarChartXAxis extends React.Component {
+    render() {
+        const labels = this.props.items.map((item) => (
+            <BarChartXAxisItem
+                {...item}
+                key={item.value} />
+        ));
+
+        return (
+            <g
+                className="bar-axis">
```

Comment:
Why are these on newlines if there's only one thing being passed to them? Is that standard or are we expecting more variables?